lib/generators/curation_concerns/install_generator.rb

```diff
@@ -104,5 +105,12 @@ def resource_types_config
     def simple_form_bootstrap
       generate 'simple_form:install --bootstrap'
     end
+
+    def simple_form_initializers
+      copy_file 'config/initializers/simple_form.rb',
```

Comment:
I was going to wait until this ticket was sorted in CC and Sufia before trying to push the change upstream. I will show the differences in comments in the code.